firmware/controllers/algo/fuel_math.cpp

```diff
@@ -72,12 +72,14 @@ DISPLAY(DISPLAY_IF(isCrankingState)) floatms_t getCrankingFuel3(float coolantTem
 	DISPLAY_SENSOR(TPS);
 	DISPLAY_TEXT(eol);
 
-	DISPLAY_TEXT(Cranking_fuel);
-	floatms_t crankingFuel = engine->engineState.DISPLAY_PREFIX(cranking).DISPLAY_FIELD(fuel) = baseCrankingFuel
```

Comment:
PLEASE never do this.  Never ever ever ever do more than one assignment on a single line.  It's impossible to read, unclear what order they might happen in, and easy to miss that this line in fact causes observable side effects IN THE MIDDLE OF THE LINE WHERE YOU CAN'T SEE IT

Reply:
fair point, fixed in master